### PR TITLE
Feat/update navbar

### PR DESCRIPTION
**Pull request by BLuEScioN**
Updated the navbar based on this issue https://github.com/hirosystems/explorer/issues/1478

### CODE REVIEW

**Comment by eugeniadigon** [comment]
Thanks Nick, looking good
Some tweaks for the font weights and colors mainly:

- On main nav bar items, `font-weight` should be regular and `font-size` should be `s`
- On dropdown items, `font-weight` should be `regular`
- On network dropdown:
  - Apply same font-weight changes (regular)
  - URL text should be `text-subdued` color, `font-weight regular`, `font-size s`

🌙 Dark mode:
- Network tag (Nakamoto) should be in [dark-theme colors](https://www.figma.com/file/31q1g27k69z1yXb6r6vrRD/Stacks-Explorer-Working-File?type=design&node-id=6625%3A7525&mode=design&t=I3Rj2TOwM8Y3zOBj-1) (font: purple-300, bg: purple-900, border: purple-700)
- Border color of dropdown should be `border-secondary` (#1C2024)

File: src/app/_components/NavBar/BtcStxPrice.tsx [hunk 1]
```diff
@@ -29,9 +29,14 @@ function PriceContainer({
 function BtcStxPriceBase({ tokenPrice }: { tokenPrice: TokenPrice }) {
   const formattedBtcPrice = tokenPrice.btcPrice ? usdFormatter.format(tokenPrice.btcPrice) : '';
   const formattedStxPrice = tokenPrice.stxPrice ? usdFormatter.format(tokenPrice.stxPrice) : '';
+  const isDesktop = useIsDesktop();
   return (
     <Flex gap={6} minWidth={'172px'}>
-      <PriceContainer icon={<Icon as={BitcoinIcon} size={'18px'} />} minWidth={'92px'}>
+      <PriceContainer
+        icon={<Icon as={BitcoinIcon} size={'18px'} />}
```
Comment (He1DAr) [inline src/app/_components/NavBar/BtcStxPrice.tsx:36]:
```suggestion
        icon={<Icon as={BitcoinIcon} size={4.5} />}
```

File: src/app/_components/NavBar/ColorModeButton.tsx [hunk 2]
```diff
@@ -1,22 +1,41 @@
...
-  const { colorMode, toggleColorMode } = useColorMode();
-  const Icon = colorMode === 'light' ? PiSunDim : PiMoon;
+export function ColorModeButton({
+  colorModeOverride,
+  ...rest
+}: {
+  colorModeOverride?: ColorMode;
+} & IconButtonProps) {
+  const { colorMode, toggleColorMode, setColorMode } = useColorMode();
+  const Icon = colorModeOverride
```
Comment (He1DAr) [inline src/app/_components/NavBar/ColorModeButton.tsx:15]: could benefit from `useMemo`

File: src/app/_components/NavBar/NetworkLabel.tsx [hunk 3]
```diff
@@ -77,22 +88,22 @@ export const NetworkLabel: FC<{ network: Network }> = ({ network }) => {
             </Badge>
           ) : (
             <Badge
-              color={'purple.600'}
-              bg={'purple.100'}
+              color={colorMode === 'light' ? 'purple.600' : 'pruple.300'}
```
Comment (He1DAr) [inline src/app/_components/NavBar/NetworkLabel.tsx:91]: We usually use `useColorModeValue` on these instances.

File: src/app/_components/NavBar/LabelWrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import { useIsDesktop } from './utils';
+
+export const LabelWrapper = ({ label, href, onClick }: NavItem) => {
+  const isDesktop = useIsDesktop();
+  return (
+    <Flex
+      alignItems={'center'}
+      borderRadius="xl"
+      width="full"
+      {...(isDesktop ? { _hover: { bg: 'hoverBackground' } } : {})}
```
Comment (He1DAr) [inline src/app/_components/NavBar/LabelWrapper.tsx:14]: You can use [Chakra UI responsive style array syntax ](https://v2.chakra-ui.com/docs/styled-system/responsive-styles#the-array-syntax)to achieve this without custom hook or conditional style

**Review by He1DAr — CHANGES_REQUESTED** [review]

File: src/app/_components/NavBar/BtcStxPrice.tsx (hunk 1 above)
Comment (BLuEScioN) [reply src/app/_components/NavBar/BtcStxPrice.tsx:36]: done

File: src/app/_components/NavBar/ColorModeButton.tsx (hunk 2 above)
Comment (BLuEScioN) [reply src/app/_components/NavBar/ColorModeButton.tsx:15]: done

File: src/app/_components/NavBar/NetworkLabel.tsx (hunk 3 above)
Comment (BLuEScioN) [reply src/app/_components/NavBar/NetworkLabel.tsx:91]: done

File: src/app/_components/NavBar/LabelWrapper.tsx (hunk 4 above)
Comment (BLuEScioN) [reply src/app/_components/NavBar/LabelWrapper.tsx:14]: done

**Review by He1DAr — APPROVED** [review]

**Pull request merged**